app/src/test/java/org/oppia/app/testing/options/AppLanguageFragmentTest.kt

```diff
@@ -0,0 +1,53 @@
+package org.oppia.app.testing.options
+
+import android.content.Intent
+import androidx.test.core.app.ActivityScenario.launch
+import androidx.test.core.app.ApplicationProvider
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.action.ViewActions.click
+import androidx.test.espresso.assertion.ViewAssertions.matches
+import androidx.test.espresso.matcher.ViewMatchers.isChecked
+import androidx.test.espresso.matcher.ViewMatchers.isRoot
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.oppia.app.R
+import org.oppia.app.options.APP_LANGUAGE
+import org.oppia.app.options.AppLanguageActivity
+import org.oppia.app.recyclerview.RecyclerViewMatcher.Companion.atPositionOnView
+import org.oppia.app.utility.OrientationChangeAction.Companion.orientationLandscape
+import org.robolectric.annotation.LooperMode
+
+@RunWith(AndroidJUnit4::class)
+class AppLanguageFragmentTest {
+
+  @Test
+  @LooperMode(LooperMode.Mode.PAUSED)
+  fun changeAppLanguageToFrench_changeConfiguration_selectedLanguageIsFrench() {
```

Comment:
Please what you are testing first

`testAppLanguage_changeAppLanguageToFrench_changeConfiguration_selectedLanguageIsFrench `

Reply:
Done